__tests__/pet.test.js

```diff
@@ -0,0 +1,232 @@
+const Pet = require('../src/pet');
+
+const minFitness = 0;
+const maxFitness = 10;
+
+const minAge = 0;
+const maxAge = 30;
+
+const minHunger = 0;
+const maxHunger = 10;
+
+const yearIncrement = 1; // adds one year to age
+const hungerIncrement = 5; // adds 5+ to hunger level when grows up
+
+const errorMsg = 'Your pet is no longer alive :(';
```

Comment:
This is better! You could even go a step further by defining an object to house them in:
```
const messages = {
  died: 'Your pet is no longer alive :(',
  walk: 'I need a walk',
  hungry: 'I am hungry',
  healthy: 'I feel great'
};
```
You could then reference `Message.walk` or so in the tests